docs/source/notebooks/GP-Circular.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<ul><li>Can you put the observed data and the <code>angles</code> object into context? This would help readers understand in which cases a circular kernel would be useful. I guess this would also help understand why you shift some variables by 4 in the model below</li><li>Would be nice to explain the conceptual difference btw <code>y_sampled</code> and <code>y_pred</code></li><li>Also, any reason you're not doing the full Bayesian sampling?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4082/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>full bayes is too slow</p>

Reply:
totally rewritten